src/components/pages/home.jsx

```diff
@@ -54,28 +57,52 @@ const HomePage = React.createClass({
       query: {q: this.state.inputValue},
       hash: `#${searchInputId}`,
     })
+    this.setState({source: "search"})
   },
   locationHasChanged(location) {
     const {router} = this.context
     const oldLocation = this.props.location
     // Check that the new location stays on the Home page, to avoid overwriting searchQuery in App state.
     if (this._isMounted && router.isActive(oldLocation)) {
       const searchQuery = location.query.q || ""
+      const sourceOfQuery = location.query.source || ""
       this.context.setSearchQuery(searchQuery)
       this.setState({inputValue: searchQuery})
+      this.setState({source: sourceOfQuery})
     }
   },
   render() {
-    const {inputValue} = this.state
+    const inputValue = this.state.inputValue
+    const source = this.state.source
     const {searchQuery, searchResults} = this.context
+    const countryPackageName = this.props.countryPackageName
+    const changelogURL = `http://www.github.com/openfisca/` + `${countryPackageName}` + `/blob/master/CHANGELOG.md`
```

Comment:
However, you should not use this : this link would not work if a new country host its country package on another github organisation.

You should instead use `gitHubProject `, which is [set in the configuration](https://github.com/openfisca/legislation-explorer/blob/master/config/production.france.js#L8).